### ISSUE TITLE
search feature

### ISSUE THREAD

**Issue by acoul** [coxley/aspath_graph#5]
is it possible to have a search feature, menu ? 

it would be great to be able to select a specific node and also be able to set the depth of the next hops for each link to be solely displayed

something similar perhaps to http://bgpmap.awmn/?nodeID=3298

**Comment by coxley** [comment]
Yep, that'd be very cool! There's probably something that could be done for that to filter js entities in the graph. 

In the current state, `aspath_graph` mostly generates the raw dataset needed for `netjson` to be happy and feeds it. Would like for this to be a bit more feature-ful in the frontend. Prettier, etc.

My workspace at home is a WIP currently but hopefully I'll be able to work on this sooner than later. :)
